### ISSUE TITLE
Paragraph not recognizing absolute file paths

### ISSUE THREAD

**Issue by JustasR1** [lewis-chinery/Paragraph#2]
Hi,

I was able to successfully clone the Paragraph git folder, create a venv, install dependencies, and run the -e examples data set. However, when I run custom data, Paragraph is unable to recognize the absolute file paths. The command fails on the first argument, the .csv file paths, returning the line 108 error code from command_line.py, indicating an argument of "none" was passed. I've ensured my files are formatted per the examples, although the error suggests that the file path is not being recognized by the program. I can provide further information if needed, I'm currently away from my desk and don't have the terminal in front of me. Thank you for any help you can provide.

Best,
Justas

**Comment by lewis-chinery** [comment]
Hi Justas,

Thank you for your interest in Paragraph!

Could you please try running the example files again but using the full custom command this time and let me know if this works e.g. similar to the below but changing `/data/localhost/chinery/Paragraph` to your install location:

`Paragraph --pdb_H_L_csv /data/localhost/chinery/Paragraph/Paragraph/example/pdb_H_L_key.csv --pdb_folder_path /data/localhost/chinery/Paragraph/Paragraph/example/pdbs/ --out_path /data/localhost/chinery/Paragraph/Paragraph/example/example_predictions.csv`

If this works then please check your previous command closely resembles this pattern. If you are still having issues then it would be helpful if you could share more details including:
- the exact Paragraph command you are using
- the head of your pdb_H_L_csv file (omitting any potentially sensitive information)
- an `ls` of the first couple of files in your pdb_folder_path (omitting any potentially sensitive information)

Best,
Lewis

**Comment by JustasR1** [comment]
Hi Lewis,

Thank you for the response. I went through the steps you outlined but continued to get the error. However, after checking my venv environment, I noticed it was using a later version of python than 3.8.3. After rebuilding the environment using 3.8, I stopped seeing the error, so my guess is that something about how paths are passed in each version is not compatible. Unfortunately, I won't be using paragraph anymore (I figured out IMGT numbering isn't compatible with my plans), but thank you again for the help!

Best,
Justas
